### ISSUE TITLE
Inappropriate jargon in alt text for hero image on front page

### ISSUE THREAD

**Issue by cjwatson** [canonical-websites/build.snapcraft.io#874]
The alt text for the hero image on the front page is "Snapcraft hero image, push to GitHub, built automatically, release for your users."

"Hero image" is design jargon, and it does not belong in alt text whose primary audience should be users with accessibility needs.  The text of this attribute should be rephrased with consideration for those users.

**Comment by matthewpaulthomas** [comment]
The rest of the `alt` text is redundant too: it would be weird to see/hear
> Push to GitHub, built automatically, release for your users.
> - Push to GitHub
> - Built automatically
> - Released for your users

Since the image is just an illustration of the text below, it should have `alt=""`.

A quick review of `alt` text on the rest of the page:

- Wordmark: `alt="Snapcraft.io"`. Good.

- “Publish your software for” logos: `alt` missing. Should be the name of each OS, comma-separated.

- Workflow diagram: `alt="Snapcraft workflow diagram."` If you can’t see the image, that’s a non-sequitur. Here the meaningful text in the image isn’t repeated elsewhere, so we should include it in the `alt`, ideally also conveying that it’s a flow. Something like: `alt="1: You receive a pull request on GitHub. 2: Test with Travis or other CI system. 3: The code lands on your GitHub master. 4: Snapcraft builds a new snap version. 5: Auto-released to the snap store for testing. 6: You promote to beta, candidate, or stable."`.

- “Fast to install…” illustration: `alt` missing. Since it doesn’t add meaning, should have `alt=""`.

- Testimonial logos: `alt="logo"`. Non-sequitur. Since each citation already includes the organization name, these should be `alt=""`.
